addons/sale/controllers/product_configurator.py

```diff
@@ -358,6 +358,7 @@ def _get_basic_product_information(self, product_or_template, pricelist, combina
                 'description_sale': str|False,
                 'display_name': str,
                 'price': float,
+                'website_published': bool,
```

Comment:
`website_published` can't be used in `sale`. The field is only added to `product.template` in `website_sale` and should only be used in `website_sale` and its dependencies to ensure modularity.